README.md

```diff
@@ -34,10 +34,9 @@ Sailfish OS is a mobile operating system based on open source and developed by t
 
 ## Sailfish OS
 
-* [Sailfish Wiki](https://sailfishos.org/about/) - Wiki about Sailfish OS
 * [Sailfish OS Community](https://sailfishos.org/community/) - About Sailfish OS
   community
-* [together.jolla.com](https://together.jolla.com/questions/) - The place to ask
+* [SailfishOS Forum](https://forum.sailfishos.org) - The place to ask
```

Comment:
Should Together be linked somewhere still as it may have useful information while it may start being a bit dated and I understand that threads are reopened on forum as needed?

Reply:
You can still list it I think with a note that the information maybe date.